File: 6 - Retirement Calculator/main.py
from datetime import datetime

year = datetime.now().year

age = int(input("What is your current age? "))
ret = int(input("At what age do you want to retire? "))
left = ret - age

print("You have {} years until you can retire".format(left))
print("It's {}, so you can retire in {}".format(year, year+left))

File: 6 - Retirement Calculator/C/main.c
#include<stdio.h>
#include<time.h>

int main(void){
    time_t t = time(NULL);
    struct tm tm = *localtime(&t);
    int year = tm.tm_year + 1900;

    unsigned short int agenow;
    printf("What is your current age? ");
    scanf("%hu", &agenow);

    unsigned short int retireage;
    printf("At what age do you want to retire? ");
    scanf("%hu", &retireage);

    printf("You have %d years left until you can retire\n", retireage - agenow);
    printf("It's %d, so you can retire in %d\n", year, year + (retireage - agenow));


    return 0;
}